conjure/errorwriter.go

```diff
@@ -417,14 +668,59 @@ func astErrorUnsafeParamsMethod(errorDefinition spec.ErrorDefinition, info types
 				},
 			},
 			Body: []astgen.ASTStmt{
-				statement.NewReturn(
-					expression.NewCompositeLit(
+				&statement.Assignment{
+					LHS: []astgen.ASTExpr{expression.VariableVal("_"), expression.VariableVal(unsafeParamsName)},
+					Tok: token.DEFINE,
+					RHS: expression.NewCallFunction(
+						"werror", "ParamsFromError", expression.NewSelector(expression.VariableVal(errorReceiverName), causeField)),
+				},
+				&statement.Assignment{
+					LHS: []astgen.ASTExpr{expression.VariableVal("localUnsafeParams")},
```

Comment:
For the sake of consistency/symmetry should we move this to an `unsafeParams()` method too?

Reply:
Done.